src/Controller/StudentController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class StudentController extends AbstractController
+{
+    /**
+     * @Route("/student", name="student")
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+
+    public function index(Request $request)
+    {
+        $name = $request->get('name');
```

Comment:
```suggestion
        $name = $request->get('name', '');
```

Yra gera praktika turėti standartinę reikšmę. Nes `PHP` kalboje ne visada akivaizdu, kada bus grąžinta `null`, `false`, `""` arba `0`.

Plačiau:
 * https://symfony.com/doc/current/components/http_foundation.html#accessing-request-data